### PR TITLE
ba-15 Create UI Prototype: Show all inventory items

### PR DESCRIPTION
**Pull request by lyio**
Initial draft for the inventory list with static items

### CODE REVIEW

**Comment by lyio** [comment]
@flashback2k14

File: socobo-inventory.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    }
+
+    .item-name {
+      margin-top: 20px;
+    }
+  </style>
+
+  <template>
+    <paper-material vertical layout class="inventory" elevation="1">
+      <paper-material elevation="1">
```
Comment (flashback2k14) [inline socobo-inventory.html]: please use a toolbar; custom color is possible w/ --paper-toolbar-background and --paper-toolbar-color

File: socobo-inventory.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                  isChecked="{{valueOfCheckBox}}"
+                  on-check="handleCheck"
+                  on-uncheck="handleUncheck">
+                  ></socobo-list-item>
+            </template>
+        </div>
+      </paper-material>
+    </paper-material>
+
+    <paper-fab style="position:absolute; right: 30px; bottom: 30px" icon="add"></paper-fab>
```
Comment (flashback2k14) [inline socobo-inventory.html]: please use css class!

**Comment by flashback2k14** [comment]
comments added in socobo-inventory.html.

**Comment by flashback2k14** [comment]
i.O.

**Pull request merged**